apps/frontend/src/pages/explore/explore.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import { MapProvider } from "~/libs/components/components.js";
+
+import { ExploreRoutes } from "./components/components.js";
+import styles from "./styles.module.css";
+
+const Explore = (): React.JSX.Element => {
+	return (
+		<main className={styles["main"]}>
+			<MapProvider>
+				<ExploreRoutes />
+			</MapProvider>
+		</main>
+	);
+};
```

Comment:
This may look good for the demo, but it adds way too much unnecessary logic, which we don't need, so let's remove it, and as for markers,  let's make fly to markers a default behaviour on click. In case we need to customize it, we will customize it later. Except for maintenance, another huge issue with adding unnecessary logic is that it still requires review and time for refining, while it doesn't bring value to the app

```suggestion
const Explore = (): React.JSX.Element => {
	return (
		<main className={styles["main"]}>
			<MapProvider markers={mockMarkers} />
		</main>
	);
};
```